sechub-doc/src/docs/asciidoc/sechub-quickstart-guide.adoc

```diff
@@ -20,124 +20,89 @@ The following topics are covered:
 
 * [x] Getting SecHub
 * [x] Building SecHub
-* [x] Starting SecHub in Integration Test Server mode
-* [x] Getting default passwords
-* [x] Working with the REST API and https://curl.haxx.se/[cURL]
+* [x] Starting SecHub server in Integration Test mode
+* [x] Default passwords
+* [x] Working with the REST API
+* [x] Creating a project on SecHub server
+* [x] Code scan with the SecHub client
 * [x] Stopping the server
 
 === Requirements
 
-* Java SDK
+* Java 11 SDK
```

Comment:
We are still compatible with Java 8 and will continue to do so. Once it is out later this year, well will support Java 17 as well. Java 8, 11 and 17 are all long term support versions (LTS) of the JDK.

Reply:
I'll add this information as a comment and remove the '11'.